packages/authentication/azure/package.json

```diff
@@ -36,5 +38,11 @@
   },
   "publishConfig": {
     "access": "public"
+  },
+  "devDependencies": {
+    "@vitest/coverage-v8": "^1.3.1",
```

Comment:
couldn't we simply make the dependencies workspace global instead of local to the package? I'm interested in reducing the number of dependabot PRs we'll get over time.